onnx/helper.py

```diff
@@ -446,9 +473,21 @@ def make_tensor_value_info(
     if doc_string:
         value_info_proto.doc_string = doc_string
 
+    tensor_type_proto = make_tensor_type_proto(elem_type, shape, shape_denotation)
+    value_info_proto.type.CopyFrom(tensor_type_proto)
+    return value_info_proto
+
+
+def make_tensor_type_proto(
+        elem_type,  # type: int
+        shape,  # type: Optional[Sequence[Union[Text, int]]]
+        shape_denotation=None,  # type: Optional[List[Text]]
+):  # type: (...) -> TypeProto
+    """Makes a Tensor TypeProto based on the data type and shape."""
+    value_info_proto = ValueInfoProto()
```

Comment:
Seems redundant to create a ValueInfoProto and then throw it away. I think we can just do ```tensor_type_proto = TypeProto()```, right?